src/api/2fa.c

```diff
@@ -267,6 +267,7 @@ int generateTOTP(struct ftl_conn *api)
 {
 	// Generate random secret using the system's random number generator
 	uint8_t random_secret[RFC6238_SECRET_LEN];
+	log_info("A");
```

Comment:
Why are you logging A and B?

Reply:
Forgotten debug logging to check if I really tested all the places where randomness is needed when testing the two fallback layers, thank you!